rfd/0057-automatic-aws-server-discovery.md

```diff
@@ -235,6 +235,153 @@ discovery_service:
     teleport.dev/origin: "cloud"
 ```
 
+
+### Agentless installation
+
+In addition to supporting automatic Teleport agent installation, an agentless option
+will also be supported. This mode will update the OpenSSH CA to use the Teleport CA
+without installing the full Teleport Agent.
+
+The teleport ca and host keys will be generated and then uploaded to
+the the AWS Secret store where, for each node discovered a new secret
+will be created, each secret will have the following contents
```

Comment:
Let's add a section explaining how IAM permissions need to be set up for discovery service and the nodes.